This patch is introduced to resolve an autogen failure. 

In AOSC, when generating autotool's configutration files, autogen will try to
search libtool's executable in the PATH, which returns `/bin/libtool` instead
of `/usr/bin/libtool`, leading to a failure when trying to access a file via 
relate path. This is caused by our configutration of bash-startup. Since 
modifying it may cause other problems, this patch is introduced as
a workaround.

Signed-off by: Camber Huang <camber@aosc.io>

diff autogen.sh.rejected autogen.sh -pu
--- a/autogen.sh.rejected 2020-07-12 12:00:17.000000000 +0800
+++ b/autogen.sh  2022-04-13 16:52:03.814926709 +0800
@@ -65,6 +65,7 @@ done
 
 # Much like APR except we do not prefer libtool 1 over libtool 2.
 libtoolize="`./build/PrintPath glibtoolize libtoolize glibtoolize1 libtoolize15 libtoolize14`"
+libtoolize="$(readlink -f ${libtoolize})"
 lt_major_version=`$libtoolize --version 2>/dev/null | sed -e 's/^[^0-9]*//' -e 's/\..*//' -e '/^$/d' -e 1q`
 
 if [ "x$libtoolize" = "x" ]; then